### PR TITLE
Link to API on homepage

### PR DESCRIPTION
**Pull request by JuanitoFatas**
This Pull Request adds a API link to homepage for easy access. I always go to https://rubygems.org and found no place to go to API. Also I make the i18n order same as the appearance on the page for future matainence.

Previews:

`zh-TW`

screenshot 2015-10-18 03 42 30

`zh-CN`

screenshot 2015-10-18 03 42 25

`nl`

screenshot 2015-10-18 03 42 18

`en`

screenshot 2015-10-18 03 42 14

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -142,6 +142,7 @@
               <%= link_to t('.footer.about'), page_url("about"), class: "nav--v__link--footer" %>
             <%- end %>
             <%= link_to t('.footer.help'), t(:help_url), class: "nav--v__link--footer" %>
+            <%= link_to t('.footer.api'), t(:api_url), class: "nav--v__link--footer" %>
```
Comment (arthurnn) [inline app/views/layouts/application.html.erb]: not sure if it is valuable having `api_url` as a i18n key.

Comment (JuanitoFatas) [inline app/views/layouts/application.html.erb]: I am fine with string or puting in i18n as `help_url` did.

Comment (arthurnn) [inline app/views/layouts/application.html.erb]: yeah, I guess using a string is better option.

Comment (JuanitoFatas) [inline app/views/layouts/application.html.erb]: :+1: changed.

**Comment by arthurnn** [comment]
Thanks

**Pull request merged**